### ISSUE TITLE
Issue with lanschool

### ISSUE THREAD

**Issue by porgman102** [DogeNetwork/v4#100]
I have a vague suspicion that lanschool is blocking something on your new 4.4.9 update which I cannot access on my school computer can you look into this?

**Comment by DerpmanDev** [comment]
Can you try deploying v4.4.8 and see if that issue continues?

**Comment by porgman102** [comment]
I can only use vercel and everytime I try to make a new one on 4.4.8 its either a 404 error or the stupid range error thing also I'm waiting on joining the discord I'm new to it

**Comment by porgman102** [comment]
tried deploying locally but geoforce now games will not work

**Comment by DerpmanDev** [comment]
clear cache

**Comment by porgman102** [comment]
New issue with Geoforce on website links it says cannot connect to Geoforce Now however it seems to be only on my schools wifi any thoughts?
